### PR TITLE
adding programs for Q 1,2,4,5,6,10,11,16 of assignment 3

### PR DESCRIPTION
**Pull request by RahulVarma23**
adding programs for Q 1,2,4,5,6,10,11,16 of assignment 3.
working on remaining programs.
I have considered sequence as Q1,Q2..........................Q16 for whole assignment.

### CODE REVIEW

**Comment by RahulVTechnocredits** [comment]
last program is for Q15 and not for Q12.
I have considered sequence as Q1,Q2..........................Q15 for whole assignment.

File: JavaAssignments/src/rahulV/Assignment3_Q1.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package rahulV;
+
+import java.util.Scanner;
+
+public class Assignment3_Q1 {
+
+	void largestNum() { // method to display largest number
```
Comment (mkanani13) [inline JavaAssignments/src/rahulV/Assignment3_Q1.java:7]: Ravi, Its always better to create function with parameters. take input from main method and method should accept 3 parameters. so other class can also use your function on their computed values (consider a case where other class want to find largest number but dont want to ask user for values)

File: JavaAssignments/src/rahulV/Assignment3_Q2.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		System.out.println("Enter num1: ");
+		num1 = scanner.nextInt();
+		
+		System.out.println("Enter num2: ");
+		num2 = scanner.nextInt();
+		
+		if (num1 == num2) {
+			System.out.println("Both number are Equal");
+		} else {
+			greaterNum = (num1 > num2) ? num1 : num2;
```
Comment (mkanani13) [inline JavaAssignments/src/rahulV/Assignment3_Q2.java:21]: it is always better not to write logic in separate method.

File: JavaAssignments/src/rahulV/Assignment3_Q4.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+		double num1, num2, result;
+
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Enter operator: ");
+		operator = scanner.next().charAt(0);
+		System.out.println("Enter number1 and number2: ");
+		num1 = scanner.nextDouble();
+		num2 = scanner.nextDouble();
+
+		switch (operator) {
```
Comment (mkanani13) [inline JavaAssignments/src/rahulV/Assignment3_Q4.java:18]: You should create separate function to write this logic and that function should return calculated value.

File: JavaAssignments/src/rahulV/Assignment3_Q6.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+		if (marks < 50 && marks >= 0) {
+
+			System.out.println("Student is failed");
+		} else if (marks < 75 && marks >= 50) {
+
+			System.out.println("First class");
+		} else if (marks < 100 && marks >= 75) {
+
+			System.out.println("Distinction");
+		}
```
Comment (mkanani13) [inline JavaAssignments/src/rahulV/Assignment3_Q6.java:17]: add one more condition here for invalid input. []-ve values and more then 100]

**Review by mkanani13 — APPROVED** [review]
Ravi, you are doing good. Logic is correct , just work on given comments. 
For time being I am approving your PR as nothing is blocker.

**Pull request merged**